agent/battle/data/chaldea_converter.py

```
@@ -107,27 +117,27 @@ def _flush() -> None:
-            if not isinstance(skill_idx, int) or not (1 <= skill_idx <= 3):
+            if not isinstance(skill_idx, int) or not (0 <= skill_idx <= 2):
                 continue
             target = _parse_target(action.get("options"))
 
             svt_idx = action.get("svt")
             if svt_idx is None:
                 # 御主技能
-                if skill_idx == 3 and is_order_change and replace_ptr < len(replace_members):
-                    # 换人服第 3 技能 = 换人
+                if skill_idx == 2 and is_order_change and replace_ptr < len(replace_members):
+                    # 换人服第 3 技能 = 换人（Chaldea 0-based: skill=2）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)